scripts/tooling/vercel/ignore-build-step-previews.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# This script is meant to be run from the "Ignored Build Step" in Vercel.
+
+cd "$(dirname "$0")"
+
+if [ "$VERCEL_ENV" == "preview" ]; then
+  bash ignore-build-step-base.sh
```

Comment:
Just curious as to why this needs to be a separate script

Reply:
These scripts will be used by vercel, I have to configure the usage of these 2 scripts from the vercel dashboard git integration. These scripts are to make vercel skip deploys on certain conditions, and based on the projects we have we need 2 different conditions, one restricts more than the other. The vercel bot should not trigger for production deploys (different from preview) in the explorer and dashboard because those are released manually, but apps backend and ui kit should

Reply:
We are using these scripts to restrict the vercel bot, the bot integration is set to run on every single push on every branch 🙈 which is quite intense in this monorepo, so we need these scripts to stop the craziness

Reply:
Thanks for the explanation, there was no mentions to these scripts so I was confused